.github/workflows/prepare-release.yaml

```diff
@@ -3,55 +3,19 @@ name: Prepare Release
 on:
   workflow_dispatch:
     inputs:
-      last_release:
-        description: "Previous Xtensa Rust release version"
-        required: true
-        default: "v1.82.0.3"
       next_release:
         description: "Next Xtensa Rust release version"
         required: true
-        default: "v1.82.0.3"
+        default: "v1.83.0.0"
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
 jobs:
   prepare-release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Replace Xtensa Rust versions
-        run: |
-          find . -type f -exec sed -i 's/${{ github.event.inputs.last_release }}/${{ github.event.inputs.next_release }}/g' {} \;
-
-      - name: Commit changes
-        uses: EndBug/add-and-commit@v9
-        with:
-          default_author: github_actions
-          add: "*"
-          message: "Bump Xtensa Rust version to ${{ github.event.inputs.next_release }}"
-          new_branch: build/${{ github.event.inputs.next_release }}
-
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@v7
-        with:
-          token: ${{ secrets.PAT }}
-          branch: build/${{ github.event.inputs.next_release }}
-          base: main
-          delete-branch: false
-          title: "Update Xtensa Rust version to ${{ github.event.inputs.next_release }}"
-          body: ""
-          draft: false
-
-      - uses: mad9000/actions-find-and-replace-string@5
-        id: findandreplace
-        with:
-          source: ${{ github.event.inputs.next_release }}
-          find: "v"
-          replace: ""
-
```

Comment:
Yeah I gave up, I couldn't get the actions to run because they weren't verified by github. All it does is save me from a local find and replace so I'll just do it manually.